packages/e2e-tests/specs/editor/various/multi-block-selection.test.js

```diff
@@ -148,6 +148,9 @@ describe( 'Multi-block selection', () => {
 		await clickBlockAppender();
 		await page.keyboard.type( '1' );
 		await pressKeyWithModifier( 'shift', 'Enter' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '3' );
+		await page.keyboard.press( 'ArrowUp' );
```

Comment:
This extra block is needed, otherwise we can't test a multi-selection happens or not. This test was not good, it would only insert a single block.